packages/datadog-plugin-aws-sdk/src/services/cloudwatchlogs.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const BaseService = require('./base')
+
+class CloudwatchLogsService extends BaseService {
```

Comment:
`Service` in the name is redundant.

Reply:
k, i've updated these service related classes to just be the service name